<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainMenu</class>
 <widget class="QMainWindow" name="mainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="insertOrDelete">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    /*背景颜色*/  
    background-color: rgb(221, 255, 212);   
    color:black;
      
}
QPushButton:pressed
{
    
    /*按下时字向右移动4像素*/  
    padding-left:4px;
    /*按下时字向下移动4像素*/  
    padding-top:4px;
}</string>
    </property>
    <property name="text">
     <string>插入/删除某条线路</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>输入起始站:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>输入终点站: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="startStation">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="endStation">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="leastTransfer">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    /*背景颜色*/  
    background-color: rgb(217, 253, 255);   
    color:black;
      
}
QPushButton:pressed
{
    
    /*按下时字向右移动4像素*/  
    padding-left:4px;
    /*按下时字向下移动4像素*/  
    padding-top:4px;
}</string>
    </property>
    <property name="text">
     <string>最少换乘路径</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="shortestTime">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    /*背景颜色*/  
    background-color:  rgb(217, 253, 255);   
    color:black;
      
}
QPushButton:pressed
{
    
    /*按下时字向右移动4像素*/  
    padding-left:4px;
    /*按下时字向下移动4像素*/  
    padding-top:4px;
}</string>
    </property>
    <property name="text">
     <string>最短时间路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>查询路线: </string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>401</width>
      <height>55</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>北京地铁线路查询系统</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(请勿输入任何空格或者换行!)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
